Pinta/MainWindow.cs

```diff
@@ -129,14 +129,17 @@ private void DockNotebook_TabClosed (object? sender, TabClosedEventArgs e)
 	{
 		var view = (DocumentViewContent) e.Item;
 
-		if (PintaCore.Workspace.OpenDocuments.IndexOf (view.Document) > -1) {
-			PintaCore.Workspace.SetActiveDocument (view.Document);
-			PintaCore.Actions.File.Close.Activate ();
+		if (PintaCore.Workspace.OpenDocuments.IndexOf (view.Document) <= -1)
```

Comment:
While you're in here, can you change these comparisons to `< 0` which I think is a bit more standard elsewhere in the codebase